src/08-tasks/tasks.apib

```diff
@@ -8,7 +8,7 @@ Get a list of tasks.
 
     + Attributes (object)
         + filter (object, optional)
-            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional)
+            + user_id: `f29abf48-337d-44b4-aad4-585f5277a456` (string, optional) - Returns Tasks that are assigned to this user or to a team to which this user belongs.
```

Comment:
Q: should this be `...this user or to a team...` or `...this user and to a team...`?